modules/lvms-scaling-storage-of-clusters-using-rhacm.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-scaling-storage-of-clusters-using-rhacm_{context}"]
+= Scaling up the storage of clusters by using {rh-rhacm}
+
+You can scale up the storage capacity of worker nodes on the clusters by using {rh-rhacm}.
+
+.Prerequisites
+
+* You have access to the {rh-rhacm} cluster using an account with `cluster-admin` privileges.
+* You have created an `LVMCluster` custom resource (CR) by using {rh-rhacm}.
+* You have additional unused devices on each cluster to be used by {lvms-first}.
+
+.Procedure
+
+. Log in to the {rh-rhacm} CLI using your {product-title} credentials.
+. Edit the `LVMCluster` CR that you created using {rh-rhacm} by running the following command:
++
+[source,terminal]
+----
+$ oc edit -f <file_name> -ns <namespace> <1>
+----
+<1> Replace `<file_name>` with the name of the `LVMCluster` CR.
+
+. In the `LVMCluster` CR, add the path to the new device in the `deviceSelector` field.
++
+.Example `LVMCluster` CR:
+[source,yaml]
+----
+apiVersion: policy.open-cluster-management.io/v1
+      kind: ConfigurationPolicy
+      metadata:
+        name: lvms
+      spec:
+        object-templates:
+           - complianceType: musthave
+             objectDefinition:
+               apiVersion: lvm.topolvm.io/v1alpha1
+               kind: LVMCluster
+               metadata:
+                 name: my-lvmcluster
+                 namespace: openshift-storage
+               spec:
+                 storage:
+                   deviceClasses:
+# ...
+                     deviceSelector: <1>
+                       paths: <2>
+                       - /dev/disk/by-path/pci-0000:87:00.0-nvme-1
+                       optionalPaths: <3>
+                       - /dev/disk/by-path/pci-0000:89:00.0-nvme-1
+# ...
+----
+<1> Contains the configuration to specify the paths to the devices that you want to add to the LVM volume group.
+You can specify the device paths in the `paths` field, the `optionalPaths` field, or both. If you do not specify the device paths in both `paths` and `optionalPaths`, {lvms-first} adds the supported unused devices to the LVM volume group. {lvms} adds the devices to the LVM volume group only if the following conditions are met:
+* The device path exists.
+* The device is supported by {lvms}. For information about unsupported devices, see _Devices not supported by {lvms}_ in the _Additional resources_ section.
```

Comment:
There's an in-flight PR that has some actual guidance on this, but FWIW in this doc right now, this is the only spot where "Additional resources" is being italicized. For the rest of the assembly, it's been double-quotes for both the section name and `in the "Additional resources" section`.

Reply:
For now, added double-quotes to both the section name and "Additional resources" section.
Will update it later based on the finalized guidance in the PR to add in-module references.